functions/openhab.js

```diff
@@ -430,14 +511,16 @@ function syncAndDiscoverDevices(token, success, failure) {
 
 		// Now retrieve all other items
 		(function () {
+			itemLoop:
 			for (var itemNum in items) {
 				var item = items[itemNum];
+				tagLoop:
 				for (var tagNum in item.tags) {
```

Comment:
Same here - if item.tags is an array, use `item.tags.forEach(function(tag) { ... });`